website/content/docs/jekyll.md

```diff
@@ -0,0 +1,463 @@
+
+```
+Error loading the CMS configuration
+Config Errors:
+Error: Failed to load config.yml (404)
+Check your config.yml file.
+```
+
+### Add config.yml
+
+As indicated in the error message, Netlify CMS requires a configuration file called `config.yml` at the root of the `admin` directory. Let's add it now.
+
+```
+touch config.yml
+```
+
+Go back to your browser and you should see a new set of errors.
+
+```yml
+Error loading the CMS configuration
+Config Errors:
+config should have required property 'backend'
+config should have required property 'collections'
+config should have required property 'media_folder'
+config should have required property 'media_library'
+config should match some schema in anyOf
+Check your config.yml file.
+```
+
+This is good - we're making progress. Before we really dig into the configuration for our project, we'll set the minimum configuration to satisfy Netlify CMS.
+
+- For the `backend` property we'll use the [test-repo backend](https://www.netlifycms.org/docs/authentication-backends#test-repo-backend). It will let us see the cms interface without connecting to a git repository.
+- The `media_folder` property should be set to the path where you want the cms to save images. We'll follow the advice of the [jekyll docs](https://jekyllrb.com/docs/posts/#including-images-and-resources) and create an `assets/` directory. Setting the `media_folder` property will take care of the `media_library` property error as well.
+- The `collections` property requires an array of collection objects. We'll start with a collection with only the property `name` defined to see what else is required.
+
+Copy and paste the following into `config.yml`
+
+```yml
+# config.yml
+
+backend:
+  name: test-repo
+media_folder: 'assets/'
+collections:
+  - name: 'blog'
+```
+
+Back in the browser you should see new errors related to the collection we just defined.
+
+```
+Error loading the CMS configuration
+Config Errors:
+'collections[0]' should have required property 'label'
+'collections[0]' should have required property 'files'
+'collections[0]' should have required property 'folder'
+'collections[0]' should have required property 'fields'
+'collections[0]' should match exactly one schema in oneOf
+Check your config.yml file.
+```
+
+Netlify CMS is telling us we need to define some properties of on the collection.
+
+- The `label` property is the string used to identify the collection in the cms UI. We'll set it to `"Blog"`
+- `folder` should be set to the directory containing the files we want the cms to be able to create and edit. We'll start with `"_posts/"`
+- The `fields` property requires an array of field objects. We'll start with a collection with only the property `name` defined. Setting the `fields` property will take care of the `files` property error as well.
+
+With these update, `config.yml` should look like this.
```

Comment:
We should drop all of the config warning steps and just give them a working config using the `test-repo` backend up front. The walkthrough of configuration requirements may be helpful for some, but it's out of scope for this guide, which should focus entirely on the intersection of Jekyll and Netlify CMS.

We could delete everything between this line and the `index.html` code block above, and rework this line a bit to fit that context.

Reply:
I removed setting backend to `test-repo` entirely and just added a links for setting up Identity/Git-gateway and adding the Identity widget

Reply:
I removed setting backend to `test-repo` entirely because it seemed like an unnecessary intermediary step when the next section is links to the docs for setting up Identity/Git-gateway and adding the Identity widget